cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -923,6 +926,13 @@ def _validate_seurat_convertibility(self):
             )
             self.is_seurat_convertible = False
 
+        # Seurat conversion is not supported for Visium datasets.
```

Comment:
move this block as the first check in this function, and return if `self._is_visium`. That will allow us to skip checking the X matrix, which is a more expensive operation, if we don't need to because seurat convertability is already False due to the visium check.